src/main/java/app/gpx_animator/Configuration.java

```diff
@@ -92,18 +98,20 @@
     // for JAXB
     @SuppressWarnings("unused")
     private Configuration() {
+
     }
 
     @SuppressWarnings("checkstyle:ParameterNumber")
-    private Configuration(
+    public Configuration(
```

Comment:
Why did you change this constructor to be public? This class uses the [builder pattern](https://www.vogella.com/tutorials/DesignPatternBuilder/article.html), so nobody should call the constructor directly.